internal/provider/resource_apply_module.go

```diff
@@ -162,10 +166,6 @@ func (t *applyModuleResource) Schema(_ context.Context, _ resource.SchemaRequest
 				MarkdownDescription: "Optional list of variables for the run in the target workspace.",
```

Comment:
`run_variables` can be renamed to `resolved_variables` to avoid any confusion with the `variables` field.